modules/multi-architecture-import-imagestreams.adoc

```diff
@@ -9,16 +9,10 @@
 
 On an {product-title} {product-version} cluster with multi-architecture compute machines, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list.
 
-[IMPORTANT]
+[NOTE]
 ====
-The `referencePolicy.type` field of your `ImageStream` object must be set to the `Source` type for this procedure to run successfully.
-[source,yaml]
-----
-referencePolicy:
-    type: Source 
-----
+When there are nodes with multiple architectures in your cluster, the architecture of your image must be consistent with the architecture of the node. You need to ensure that the pod is assigned to the node with the appropriate architecture and that it matches the image architecture. For more information on assigning Pods to Nodes, see link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assigning Pods to Nodes].
```

Comment:
In the last sentence, before the link, should "Pods" and "nodes" have initial caps? They aren't being used as part of a title here, so I'd say no.

I don't think initial caps should be used in the link text, either.

But maybe That's just me. :-)

Reply:
Oh yeah! Just saw that as well, thank you!